service/lotusclient/lotusclient.go

```diff
@@ -88,7 +88,7 @@ func (c *Client) HealthCheck() error {
 	}
 	ctx, cancel := context.WithTimeout(c.ctx, requestTimeout)
 	defer cancel()
-	_, err := c.c.DealsList(ctx)
+	_, err := c.c.MarketListDeals(ctx)
```

Comment:
@merlinran, this is somewhat heavy.
We can switch to `.ID(ctx)` which simply returns the peer-id of the node.
That is the lightest thing that we can use for the check.